_posts/en/newsletters/2019-10-09-newsletter.md

```diff
@@ -0,0 +1,171 @@
+  sighash flag that would make possible the [eltoo][] layer for LN was
+  [discussed][noinput thread] again on the Bitcoin-dev and Lightning-dev mailing lists.
+  After summarizing previous discussions, Christian Decker asked several
+  questions: is the idea behind the proposal useful?  (Respondents seemed
+  agreed that it was.)  Do people want mandatory chaperon signatures?
+  (Respondents seemed moderately opposed.)  Do people want mandatory output
+  tagging?  (Respondents seemed opposed, some strongly.)
+
+    In response to the question about output tagging, C-Lightning
+    contributor ZmnSCPxj [proposed][zmn internal tagging] an alternative
+    tagging mechanism that would put the tag inside the taproot
+    commitment, making it invisible unless a script-path spend was used.
+    This could allow a spender who was worried about noinput to ensure
+    they didn't pay noinput-compatible scripts---the [original goal][orig
+    output tagging] behind output tagging---but without the decrease in
+    privacy and fungibility created by output tagging.  Several people
+    seemed to express interest in this idea, although it wasn't clear
+    whether they wanted to see it as part of a proposal or they just
+    preferred it to external output tagging (which, as noted above, was
+    generally opposed by respondents).
+
+    The entire thread is more than 20 messages at present and started a
+    [spin-off discussion about `OP_CAT`][cat spinoff].  Hopefully the
+    discussion will be able to settle the major unresolved issues
+    related to noinput and help get this proposal on track for
+    inclusion in a subsequent soft fork.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #13716][] adds `-stdinrpcpass` and
+  `-stdinwalletpassphrase` parameters to
+  `bitcoin-cli` that allow it to read either an RPC or wallet
+  passphrase from the standard input buffer rather than as a CLI
+  parameter that would be stored in shell history.  Echoing is also
+  disabled on stdin during reading so that the passphrase isn't visible
+  to anyone watching your screen.
+
+- [Bitcoin Core #16884][] switches the default address type
+  for users of the RPC interface (including via `bitcoin-cli`) from
+  P2SH-wrapped P2WPKH to native segwit (bech32) P2WPKH.  This is on the
+  master development code branch and is not expected to be released
+  until Bitcoin Core 0.20.0 sometime in mid-2020.  However, a [previous
+  change][gui bech32] expected to be released as part of 0.19.0 in the
+  next month or so will switch the default address type for GUI users to
+  also use bech32 P2WPKH.
+
+- [Bitcoin Core #16507][] fixes an [issue][bitcoin core #16499] where a
```

Comment:
I cannot see @jnewbery comment here, but from e-mail:

> I don't think this is exactly correct. The behaviour before this change was that any feefilter value that wasn't a multiple of 1000 could cause an inconsistency between the mempool acceptance policy (which would allow transaction to enter the mempool if it had a fee rate > than minrelaytxfee) and the relay policy (which would round the feerate down and therefore not relay if the fee rate < (minrelaytxfee rounded up to next 1000)).

specifically it was because the feefilter was using one calculation method(which made the fee seem unusable to the peer) and not sent, due to this `%1000 != 0` issue

Reply:
it seems I have achieved time travel